src/job_cache/job_cache.cpp

```diff
@@ -320,10 +320,15 @@ wcl::result<FindJobResponse, FindJobError> Cache::read_impl(const FindJobRequest
   return wcl::result_value<FindJobError>(FindJobResponse(json));
 }
 
+static int misses_from_failure = 0;
+
 FindJobResponse Cache::read(const FindJobRequest &find_request) {
+  if (misses_from_failure > timeout_config.max_misses_from_failure) {
```

Comment:
Are the two new guards supposed to also include `miss_on_failure`? I guess if `miss_on_failure` is not set we wouldn't expect `misses_from_failure` to be non-zero and therefore these guards should never trigger, so its just a style question not a functionality quesiton.

Reply:
Yeah its never going to be reached. We only use `miss_on_failure` to determine if the miss is a failure or not